styleguide/derek/examples/solutions.ejs

```diff
@@ -98,7 +98,10 @@
 
 <%- partial('../_partials/solutions/blog-feed.ejs') %>
 
-<!-- Text -->
+<div class="container standard-padding-full">
```

Comment:
Would you mind placing some sort of comment in the HTML here, as it will be helpful to distinguish between sections.